azure-pipelines.yml

```diff
@@ -120,6 +120,7 @@ jobs:
       - template: ci/report-end.yml
 
   - job: macOS
+    demands: assignment -equals broken
```

Comment:
This needs to go in the pool not in the job.

Reply:
thanks